src/app/core/data-services/api-cw20-token.service.ts

```diff
@@ -133,17 +133,22 @@ export class ApiCw20TokenService {
 
   parseUSDCToken(token, coinsMarkets) {
     const USDCMarket = coinsMarkets?.find((item) => item.coinId === USDC_COIN_ID);
-
+    const value = new BigNumber(token?.balance).multipliedBy(Number(USDCMarket?.currentPrice || 0));
```

Comment:
Sao chỗ này lại chuyển BigNumber xong ở dưới lại toNumber() ????

Reply:
Vì logic cũ em đang thấy compare với number. Nếu không to number thì nó là object thì hiển thị sai ạ.
![image](https://github.com/aura-nw/aurascan/assets/71311738/d3356e3e-c3bd-4135-b3b2-23a90629accd)

Reply:
![image](https://github.com/aura-nw/aurascan/assets/71311738/e4eb9110-12e6-4ae6-b965-69ed7cc0773b)

Reply:
Thế ở trên dùng Number luôn, chứ sao phải chuyển qua chuyển lại?

Reply:
> Thế ở trên dùng Number luôn, chứ sao phải chuyển qua chuyển lại?

Em sợ nhân vào số lớn quá ý ạ

Reply:
Số lớn quá lúc em toNumber() nó lại overflow?

Reply:
Em update done ạ.